benchmark/runner/mini_runners.cpp

```diff
@@ -0,0 +1,1161 @@
+      accessor->SetTablePointer(tbl_oid, tmp_table);
+      txn_manager_->Commit(txn, transaction::TransactionUtil::EmptyCallback, nullptr);
+    }
+
+    std::stringstream query;
+    query << "INSERT INTO tmp_table VALUES ";
+
+    std::mt19937 generator{};
+    std::uniform_int_distribution<int> distribution(0, INT_MAX);
+    for (uint32_t idx = 0; idx < num_rows; idx++) {
+      query << "(";
+      for (uint32_t i = 1; i <= num_cols; i++) {
+        query << distribution(generator);
+        if (i != num_cols) {
+          query << ",";
+        } else {
+          query << ") ";
+        }
+      }
+
+      if (idx != num_rows - 1) {
+        query << ", ";
+      }
+    }
+
+    brain::PipelineOperatingUnits units;
+    brain::ExecutionOperatingUnitFeatureVector pipe0_vec;
+    pipe0_vec.emplace_back(brain::ExecutionOperatingUnitType::INSERT, num_rows, 4 * num_cols, num_cols,
+                           static_cast<double>(num_rows));
+    units.RecordOperatingUnit(execution::pipeline_id_t(0), std::move(pipe0_vec));
+
+    uint64_t elapsed_ms;
+    {
+      common::ScopedTimer<std::chrono::milliseconds> timer(&elapsed_ms);
+      metrics_manager_->RegisterThread();
+      BenchmarkSqlStatement(qid, query.str(), &units, std::make_unique<optimizer::TrivialCostModel>(), true);
+      metrics_manager_->Aggregate();
+      metrics_manager_->UnregisterThread();
+    }
+
+    // Drop the table
+    {
+      auto txn = txn_manager_->BeginTransaction();
+      auto accessor = catalog_->GetAccessor(common::ManagedPointer(txn), db_oid);
+      accessor->DropTable(tbl_oid);
+      txn_manager_->Commit(txn, transaction::TransactionUtil::EmptyCallback, nullptr);
+    }
+
+    state.SetIterationTime(static_cast<double>(elapsed_ms) / 1000.0);
+
+    auto gc = db_main->GetStorageLayer()->GetGarbageCollector();
+    gc->PerformGarbageCollection();
+    gc->PerformGarbageCollection();
+  }
+
+  state.SetItemsProcessed(num_rows);
+}
+
+/**
+ * Arg: <0, 1>
+ * 0 - Number of columns
+ * 1 - Number of rows to insert
+ */
+BENCHMARK_REGISTER_F(MiniRunners, InsertRunners)
+    ->Unit(benchmark::kMillisecond)
+    ->UseManualTime()
+    ->Iterations(1)
+    ->Apply(GenInsertArguments);
+
+// NOLINTNEXTLINE
+BENCHMARK_DEFINE_F(MiniRunners, UpdateRunners)(benchmark::State &state) {
+  auto num_cols = state.range(0);
+  auto num_rows = state.range(1);
+
+  execution::query_id_t qid;
+  switch (num_cols) {
+    case 1:
+      qid = UPDATE_COL_1_QID;
+      break;
+    case 4:
+      qid = UPDATE_COL_4_QID;
+      break;
+    case 8:
+      qid = UPDATE_COL_8_QID;
+      break;
+    case 15:
+      qid = UPDATE_COL_15_QID;
+      break;
+    default:
+      UNREACHABLE("Unexpected number of columns for UpdateRunners");
+      break;
+  }
+
+  // NOLINTNEXTLINE
+  for (auto _ : state) {
+    // Create temporary table schema
+    std::stringstream query;
+    query << "UPDATE "
+          << "INTEGERCol15Row" << num_rows << "Car1 SET ";
```

Comment:
Extract a table name generation static function in `TableGenerator` that takes in the type, the row number, the column number, and the cardinality. And use the same function both in the `TableGenerator` and all the table name references here.